### ISSUE TITLE
Traceback error

### ISSUE THREAD

**Issue by 82n** [znqi/discordtemplate#2]
## What happened
* running `discord create <BOT_NAME>` gets error 

## Output
```python
[korea@fedora ~]$ discord create shiro
Traceback (most recent call last):
  File "/home/korea/.local/bin/discord", line 8, in <module>
    sys.exit(run())
  File "/home/korea/.local/lib/python3.9/site-packages/discord_template/__main__.py", line 232, in run
    startup()
  File "/home/korea/.local/lib/python3.9/site-packages/discord_template/__main__.py", line 50, in startup
    if data['auto_update'] == True:
UnboundLocalError: local variable 'data' referenced before assignment
```

**Comment by znqi** [comment]
That error was probably because of the startup file is missing? Try to create `startup` file on the module directory and type 

```json
{"auto_update": true}
```

**Comment by znqi** [comment]
Ah btw, i'll fix that soon

**Comment by znqi** [comment]
Fixed already, you may now install **v1.1.3** as well as improved installation performance by removing requests and use
`urllib.request` instead. Thank you so much for your issue. If don't have any question, you may now close this issue.
